frontend/e2e-tests/MoEmployeeAssociationTab.js

```diff
@@ -0,0 +1,90 @@
+import VueSelector from 'testcafe-vue-selectors'
+import { Selector } from 'testcafe'
+import { baseURL } from './support'
+
+let moment = require('moment')
+
+fixture('MoEmployeeAssociationTab')
+  .page(`${baseURL}/medarbejder/554ef7cc-4a18-478f-865f-3c2f0dd20a69`)
+
+const dialog = Selector('.modal-content')
+
+// Association
+const parentAssociationInput = dialog.find('input[data-vv-as="Angiv enhed"]')
+
+const associationTypeSelect = dialog.find('.select-association select[data-vv-as="Tilknytningsrolle"]')
+const associationTypeOption = associationTypeSelect.find('option')
+
+const fromDateInput = dialog.find('.from-date .form-control')
+
+const submitButton = dialog.find('button .btn .btn-primary')
+
+test('Workflow: employee association tab', async t => {
+  let today = moment()
+
+  await t
+    .setTestSpeed(0.8)
```

Comment:
The tests shouldn't rely on setting the speed; rather, whomever is running the test manually should pass `--speed` to TestCafé if when needed.

(Our tests are already slow enough; this just makes them slower.)

Reply:
:thumbsup: